keps/sig-storage/4547-in-tree-volume-plugins-remove-dynamic-provisioning-support/README.md

```diff
@@ -0,0 +1,854 @@
+<!--
```

Comment:
Please add a `keps/prod-readiness/sig-storage/4547.yaml` file and assign to me for alpha:

```
kep-number: 4547
alpha:
  approver: "@jpbetz"
```